ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue

```diff
@@ -162,11 +167,25 @@ export default defineComponent({
       localState.dataLoaded = true
     })
 
+    const { buildApiData } = useNewMhrRegistration()
+
+    const submit = async () => {
+      // TODO: for testing/demo purpose, DELETE AFTERWARDS
+      setValidation(MhrSectVal.REVIEW_CONFIRM_VALID, MhrCompVal.VALIDATE_APP, true)
+      if (localState.validateMhrRegistration) {
+        localState.submitting = true
+        const result = await submitMhrRegistration(buildApiData())
+        localState.submitting = false
+        console.log(result)
```

Comment:
remove dev code

Reply:
I kept it for QA testing.

Reply:
I don't think QA will need this. 
If a developer wants to see the `result` in the browser, you can get it from the network request response.

Reply:
that's true, removed.